### DISCUSSION TITLE
How does AutoCrop work now?

### DISCUSSION THREAD

**Discussion by SnorreSelmer** [staxrip/staxrip#1526] [Q&A] [answered]
So, I used to just encode videos, and they were auto-cropped, but that doesn't seem to happen anymore. If I open a new project, the Crop is unchecked every time, and if I check it, it "crops" to `(0, 0, -0, -0)`...

I thought it might be an issue with old Templates, so I deleted them all and started fresh. Didn't help...
![bilde](https://github.com/user-attachments/assets/e86a3b36-19ab-4927-bbe8-a5674855a743)

**Comment by Dendraspis** [comment, marked as answer]
> I thought it might be an issue with old Templates, so I deleted them all and started fresh.

That was a good decision. Even it didn't help you with this problem, it will prevent other issues.
But I recommend checking the settings and project options...

Extract from the Changelog:

```
v2.40.0 (2024-06-08)
- Crop: Move AutoCrop settings from Settings to Project Options
```

Here:
<img width=500 src=https://github.com/user-attachments/assets/7b5f9584-c00e-40ec-a187-a4c0e2666104>

**Reply by SnorreSelmer** [reply]
Aha! I looked all over for this, but I apparently didn't find the _Project_ Options.. 😳 
All is good, and thanks for the help!
